client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx

```diff
@@ -46,6 +47,18 @@ const BusinessPlanDetails = ( { selectedSite, sitePlans, selectedFeature } ) =>
 					href={ '/themes/' + selectedSite.slug } />
 			}
 
+			{ ! selectedFeature && isEnabled( 'manage/plugins/upload' ) &&
+				<PurchaseDetail
+					icon="plugins"
+					title={ i18n.translate( 'Add a Plugin' ) }
+					description={ i18n.translate(
+						'Search and add plugins right from your dashboard, or upload a plugin ' +
+						'from your computer with a drag-and-drop interface.'
+					) }
+					buttonText={ i18n.translate( 'Upload a plugin now' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **68** times:
`translate( 'Upload Plugin' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).